src/threads_and_async.md

```diff
@@ -101,7 +101,7 @@ This can sometimes be problematic, for example when it comes to widgets.
 Fortunately, the [`spawn_local`](https://relm4.org/docs/next/relm4/fn.spawn_local.html) function allows us to spawn local futures, which don't require `Send` because they run on the main thread.
 This works because GTK uses an event loop from GLib to handle asynchronous events, which also allows you to execute futures.
 
-The only drawback of this solution is that not all async libraries are fully compatible with the GLib executor, since they must use Tokyo.
+A drawback of this solution is that you only can use libraries which support Tokio, as the GLib executor depends on it.
```

Comment:
Actually, Glib doesn't depend on tokio, but many async crates do. For example, using reqwest in future on a GLib executor will panic because it needs some tokio-specific features. This is not obvious for beginners because it should be as simple as using async/await but sadly that's not true in reality.